src/homework3/task2.py

```diff
@@ -0,0 +1,29 @@
+""" List practice
+
+1. Используйте генератор списков чтобы получить следующий: ['ab', 'ac', 'ad', 'bb', 'bc', 'bd'].
+2. Используйте на предыдущий список slice чтобы получить следующий: ['ab', 'ad', 'bc'].
+3. Используйте генератор списков чтобы получить следующий ['1a', '2a', '3a', '4a'].
+4. Одной строкой удалите элемент  '2a' из прошлого списка и напечатайте его.
+5. Скопируйте список и добавьте в него элемент '2a' так
+чтобы в исходном списке этого элемента не было.
+"""
+
+# 1 Create list by generator
+lst = [str_1 + str_2 for str_1 in ['a', 'b'] for str_2 in ['b', 'c', 'd']]
```

Comment:
'bcd' ?

Reply:
also we can write ...in 'ab' for str_2 in 'bcd'